src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-  <Component
-    :is="isBoxed ? 'VLink' : 'VWarningSuppressor'"
+  <VLink
+    v-bind="containerAttributes"
     class="audio-track group"
     :aria-label="ariaLabel"
     role="region"
-    v-bind="layoutBasedProps"
+    :href="`/audio/${audio.id}`"
     @keydown.native.shift.tab.exact="$emit('shift-tab', $event)"
-    @keydown.native.space="handleSpace"
+    @keydown.native.exact="handleKeydown"
```

Comment:
What is the purpose of `exact` modifier here? I thought it only works if you add the key modifiers (just like the `.shift.tab.exact` above (there, it's only called when both keys were pressed together).

Reply:
Yeah, this `.exact` was required by the linter. Apparently the presence of `.exact` for the `@keydown` event above means all other `@keydown` listeners must have `.exact` specified.